lib/hanami/helpers/html_helper/empty_html_node.rb

```diff
@@ -124,7 +124,11 @@ def boolean_attribute(attribute_name, _value)
         end
 
         def attribute(attribute_name, value)
-          %(#{ATTRIBUTES_SEPARATOR}#{attribute_name}="#{value}")
+          if attribute_name.to_sym == :value
+            %(#{ATTRIBUTES_SEPARATOR}#{attribute_name}="#{Hanami::Utils::Escape.html(value)}")
+          else
+            %(#{ATTRIBUTES_SEPARATOR}#{attribute_name}="#{value}")
```

Comment:
Could we simply make a list of the attributes that need escaping? (like `value`, `title`, ...)